pkgs/applications/misc/bazecor/default.nix

```diff
@@ -17,34 +13,40 @@ appimageTools.wrapAppImage rec {
     # Workaround for https://github.com/Dygmalab/Bazecor/issues/370
     postExtract = ''
       substituteInPlace \
-        $out/usr/lib/bazecor/resources/app/.webpack/main/index.js \
-        --replace \
+        $out/usr/lib/${pname}/resources/app/.webpack/main/index.js \
+        --replace-fail \
           'checkUdev=()=>{try{if(c.default.existsSync(f))return c.default.readFileSync(f,"utf-8").trim()===l.trim()}catch(e){console.error(e)}return!1}' \
           'checkUdev=()=>{return 1}'
     '';
   };
+in appimageTools.wrapAppImage {
+  inherit pname version src;
 
   # also make sure to update the udev rules in ./10-dygma.rules; most recently
   # taken from
   # https://github.com/Dygmalab/Bazecor/blob/v1.3.11/src/main/utils/udev.ts#L6
 
-  extraPkgs = p: (appimageTools.defaultFhsEnvArgs.multiPkgs p) ++ [
-    p.glib
-  ];
+  extraPkgs = p: (appimageTools.defaultFhsEnvArgs.multiPkgs p) ++ [ p.glib ];
 
   # Also expose the udev rules here, so it can be used as:
   #   services.udev.packages = [ pkgs.bazecor ];
   # to allow non-root modifications to the keyboards.
 
   extraInstallCommands = ''
-    install -m 444 -D ${src}/Bazecor.desktop -t $out/share/applications
-    substituteInPlace $out/share/applications/Bazecor.desktop \
-      --replace 'Exec=Bazecor' 'Exec=bazecor'
+    ln -s $out/bin/${pname}-${version} $out/bin/${pname}
```

Comment:
Is this symlink here necessary? It fails to build for me, but it works if I remove it

Reply:
Interesting, I didn't get any errors last time I built it locally.
I will change `ln` to `mv` and won't run into this issue anymore

Reply:
@gvolpe I think this PR has been gathering dust for too long and resulted in this breakage. 

`appimageTools` no longer creates a binary `$out/bin/${name}` if `pname` and `version` are provided. See #271071.

It used to be needed at the time I wrote this PR. I updated the PR to reflect these changes.